sample_metadata/parser/generic_metadata_parser.py

```diff
@@ -303,15 +303,15 @@ async def check_files_covered_by_rows(
 
         filenames: List[str] = sum(await asyncio.gather(*filename_promises), [])
         fs = set(f.strip() for f in filenames if f and f.strip())
-        relevant_extensions = ('.cram', '.fastq.gz', '.bam')
+        relevant_extensions = ('.cram', '.crai', '.fq.gz', '.fastq.gz', '.bam', '.bai')
```

Comment:
Oh you want to tell if an index is missing too? We don't require indexes for other ingested data, so happy for this change if you can parameterise it.

Reply:
Ah, I guess we don't have to check them at all. I initially thought the error was because the index is not listed here, but it was because of the sets logic below. I will remove bai/crai.